src/shared/bus-util.c

```diff
@@ -1127,7 +1127,7 @@ static int map_basic(sd_bus *bus, const char *member, sd_bus_message *m, unsigne
                 if (r < 0)
```

Comment:
Please add an explanation to the commit message why this change is appropriate, including the effect on the callers. This is definitely not a trivial change!